package.json

```diff
@@ -18,11 +18,14 @@
     "@supabase/supabase-js": "^2.39.2",
     "@t3-oss/env-nextjs": "^0.7.1",
     "@vercel/analytics": "^1.2.2",
+    "copy-text-to-clipboard": "^3.2.0",
```

Comment:
web api supports this natively so no need library: https://www.freecodecamp.org/news/copy-text-to-clipboard-javascript/

Reply:
done

Reply:
i used that for simplicity. np, changed